src/getAll.js

```diff
@@ -1,23 +1,41 @@
 import ColorSpace from "./ColorSpace.js";
 import getColor from "./getColor.js";
+import { toPrecision, type } from "./util.js";
 
 /** @typedef {import("./types.js").ColorTypes} ColorTypes */
 /** @typedef {import("./types.js").Coords} Coords */
 
 /**
  * Get the coordinates of a color in any color space
  * @param {ColorTypes} color
- * @param {string | ColorSpace} space The color space to convert to. Defaults to the color's current space
+ * @param {string | ColorSpace | object} [options=color.space] If a string or ColorSpace, the color space to convert to. Defaults to the color's current space.
+ *                                       If an object:
+ *                                       [options.space=color.space] — the color space to convert to (defaults to the color's current space);
+ *                                       [options.precision] — the number of significant digits to round the coordinates to.
```

Comment:
It doesn’t seem to me like TypeScript is properly understanding the type here. Maybe making a `@typedef` for the function’s options would be best? I can add a code suggestion for that you agree

Reply:
Yes I think a typedef would make sense here. Though I thought we had found a way to do overloads?